MapleServer2/Network/PacketRouter.cs

```diff
@@ -31,7 +30,6 @@ public class PacketRouter<T> where T : Session {
 
         private void Register(ImmutableDictionary<RecvOp, IPacketHandler<T>>.Builder builder,
                 IPacketHandler<T> packetHandler) {
-            logger.Debug($"Registered {packetHandler}");
```

Comment:
Why removing this?

Reply:
Just creates noise in the console on every start up for me, doesn't seem like information you need to know every time the server starts, the handler will register if you follow the structure of all the other handlers